Formula/node.rb

```diff
@@ -38,8 +38,8 @@ class Node < Formula
   # We will accept *important* npm patch releases when necessary.
   # https://github.com/Homebrew/homebrew/pull/46098#issuecomment-157802319
   resource "npm" do
-    url "https://registry.npmjs.org/npm/-/npm-3.10.3.tgz"
-    sha256 "82e6089077d40370da445b9209557ee03e3685c8d23dae9cf7a9b1635c7824de"
+    url "https://registry.npmjs.org/npm/-/npm-3.10.7.tgz"
```

Comment:
Are there any specific reasons to update `npm` to `3.10.7`? AFAIK the upstream Node 6.6.0 releases bundles `npm` 3.10.3